app/models/qa/iri_template/url_config.rb

```diff
@@ -0,0 +1,56 @@
+# Provide access to iri template configuration.
+module Qa
+  module IriTemplate
+    class UrlConfig
+      TYPE = "IriTemplate".freeze
+      CONTEXT = "http://www.w3.org/ns/hydra/context.jsonld".freeze
+      attr_reader :template # [String] the URL template with variables for substitution (required)
+      attr_reader :variable_representation # [String] always "BasicRepresentation" # TODO what other values are supported and what do they mean
+      attr_reader :mapping # [Array<Qa::IriTempalte::VariableMap>] array of maps for use with a template (required)
+
+      # @param [Hash] url_template configuration hash for the iri template
+      # @option url_template [String] :template the URL template with variables for substitution (required)
+      # @option url_template [String] :variable_representation always "BasicRepresentation" # TODO what other values are supported and what do they mean
```

Comment:
Or is this supposed varriable_representation?

Reply:
The IRI Template code doesn't yet use any values other than BasicRepresentation, so although this is listed in the options, it isn't processed at this time.